src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -585,9 +585,9 @@ template <typename T, int N>
 class CovarianceConversionReader : public FitsColumnReader {
 public:
     static std::string guessUnits(std::string const &oldUnits) {
-        static boost::regex const regex("(.*)(\\^(\\d+))?", boost::regex::perl);
-        boost::smatch m;
-        if (!boost::regex_match(oldUnits, m, regex)) {
+        static std::regex const regex("(.*)(\\\\^(\\d+))?");
```

Comment:
Is this right?  I thought the `^` caret was being escaped.  If it is, a comment would help future readers.

Reply:
This was wrong. Should indeed be `\\^`